src/ucs/algorithm/levenshtein_dist.c

```diff
@@ -13,36 +13,37 @@
 #include <ucs/sys/sys.h>
 #include <string.h>
 
-#define MAX_ENV_VAR_LENGTH 30
+#define MAX_UCX_ENV_VAR_LENGTH 30

```

Comment:
It has nothing to do with "ENV" (in theory this func can be used for another purpose in the future).